Sources/Public/Animation/LottieView.swift

```diff
@@ -175,9 +175,20 @@ public struct LottieView<Placeholder: View>: UIViewConfiguringSwiftUIView {
     return copy
   }
 
-  /// Returns a copy of this view that loops its animation whenever visible
+  // Returns a copy of this view playing once from the current frame to the end frame
+  public func play() -> Self {
+    play(loopMode: .playOnce)
+  }
+
+  /// Returns a copy of this view that loops its animation from the start to end whenever visible
   public func looping() -> Self {
-    play(.fromProgress(0, toProgress: 1, loopMode: .loop))
+    play(loopMode: .loop)
```

Comment:
changing this back since as-is this was unintentionally a behavior change

```suggestion
    play(.fromProgress(0, toProgress: 1, loopMode: .loop))
```